layers/meta-resin-allwinner/recipes-bsp/u-boot/u-boot-neutis/boot.cmd

```diff
@@ -0,0 +1,8 @@
+# Copied from meta-resin-allwinner/recipes-bsp/u-boot/files/0001-Add-Resin-specific-boot-command.patch
+
+setenv resin_kernel_load_addr ${kernel_addr_r};
+run resin_set_kernel_root;
+setenv bootargs ${resin_kernel_root} rootfstype=ext4 rootwait console=$console;
+fatload ${resin_dev_type} ${resin_dev_index}:${resin_boot_part} ${resin_kernel_load_addr} Image;
+fatload ${resin_dev_type} ${resin_dev_index}:${resin_boot_part} ${fdt_addr_r} dtb/${fdtfile};
+booti ${resin_kernel_load_addr} - ${fdt_addr_r}
```

Comment:
Why can't this be patched like for other boards? What is the problem with 64 bits u-boot?
Just as a side note, here's how we do it for other boards. https://github.com/balena-os/resin-artik710/blob/master/layers/meta-resin-artik710/recipes-bsp/u-boot/patches/0007-artik710_raptor.h-Board-specific-integration-with-re.patch (this also happens to be 64 bits)